bin/gwdetchar-slow-correlation

```diff
@@ -295,6 +300,7 @@ def process_channel(input_,):
 
 pool = multiprocessing.Pool(nprocplot)
 results = pool.map(process_channel, auxdata.iteritems())
+#results = pool.map(partial(process_channel, thresh=threshold), auxdata.iteritems())
```

Comment:
they are not needed they were a product of me trying to clean up the commits, and i cannot remove yet through git.